index.js

```diff
@@ -6,19 +6,31 @@ var fs        = require('fs')
   , envWriter = require('env-writer')()
   , envData
 
-module.exports = function (envFile) {
+module.exports = function (envFile, callback) {
+  if (typeof envFile === 'function') {
+    callback = envFile;
+    envFile = undefined;
+  }
+
+  if (!callback) {
+    callback = function() { /* noop */ }
+  }
 
   if (typeof envFile === 'undefined')
-    envFile = '.env'
+    envFile = '.env';
 
   try {
-    envData = fs.readFileSync(envFile)
+    envData = fs.readFileSync(envFile);
```

Comment:
Might be good since we are adding callbacks to make it so when a callback is used, createReadStream is used in place of readFileSync and then piped into envReader. Thoughts?